turbo/consumers.py

```diff
@@ -4,7 +4,9 @@
 from django.template.loader import render_to_string
 from django.core.signing import Signer, BadSignature
 
-from turbo import make_channel_name
+
+class TurboStreamException(BaseException):
```

Comment:
Nice! Having our own exceptions will help with debugging. Small note, it's usually recommended to subclass `Exception` instead: https://docs.python.org/3/library/exceptions.html#base-classes